src/pages/SignUpVolunteer.js

```diff
@@ -40,13 +46,21 @@ function SignUpResearcher() {
     return errors;
   };
 
-  const handleSubmit = values => {
-    //TODO: authenticate to allow login
-
-    // Db.addVolunteerData(useContext(UserContext)?.uid, ...); TODO
-    // navigate to next page
-
+  const handleSubmit = async values => {
+    try {
+      await Db.addVolunteerData(userId, values).then;
```

Comment:
Similar issue as the previous PR, committed a fix for it and will explain it in that!